morphic.js

```diff
@@ -7682,19 +7682,21 @@ MenuMorph.prototype.unselectAllItems = function () {
     this.changed();
 };
 
-MenuMorph.prototype.popup = function (world, pos) {
+MenuMorph.prototype.popup = function (world, pos, submenu) {
```

Comment:
Well, I agree that a comment (which I added) helps.  But I disagree about the two-procedures.  Abstraction!  With two procedures, when you change one of them and forget to change the other, things will go wrong.